### DISCUSSION TITLE
not equal option

### DISCUSSION THREAD

**Discussion by KezHalls** [Snow-Shell/servicenow-powershell#220] [Q&A] [answered]
Hi,
Great module.  been using for a year or more now and just updated to latest version.

Is there an option to filter on a "not like" or "not equal" to

Thanks
Kez

**Comment by gdbarron** [comment, marked as answer]
Yep, use -ne or -notlike for your comparison operator just like powershell.  See `$script:ServiceNowOperator` for a list of all operators.
